ts/features/bonus/cgn/components/CgnCardInWalletComponent.tsx

```diff
@@ -1,43 +1,93 @@
+import { View } from "native-base";
 import { navigateToCgnDetails } from "../navigation/actions";
 import { isCgnInformationAvailableSelector } from "../store/reducers/details";
 import { cgnDetails } from "../store/actions/details";
 import { useActionOnFocus } from "../../../../utils/hooks/useOnFocus";
 import { GlobalState } from "../../../../store/reducers/types";
-import { H5 } from "../../../../components/core/typography/H5";
-import { IOColors } from "../../../../components/core/variables/IOColors";
+import cgnBackground from "../../../../../img/bonus/cgn/cgn-preview.png";
+import cgnLogo from "../../../../../img/bonus/cgn/cgn_logo.png";
+import TouchableDefaultOpacity from "../../../../components/TouchableDefaultOpacity";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import { H3 } from "../../../../components/core/typography/H3";
+import TypedI18n from "../../../../i18n";
 
 type Props = ReturnType<typeof mapDispatchToProps> &
   ReturnType<typeof mapStateToProps>;
 
+const styles = StyleSheet.create({
+  preview: {
+    marginBottom: -20,
+    height: 88
+  },
+  imagePreview: {
+    resizeMode: "stretch",
+    height: 88,
+    width: "100%"
+  },
+  paddedContentPreview: {
+    paddingLeft: 18,
+    paddingBottom: 10,
+    paddingRight: 22
+  },
+  spaced: {
+    justifyContent: "space-between"
+  },
+  previewLogo: {
+    resizeMode: "contain",
+    height: 40,
+    width: 40
+  },
+  upperShadowBox: {
+    marginBottom: -13,
+    borderRadius: 8,
+    borderTopWidth: 13,
+    borderTopColor: "rgba(0,0,0,0.1)",
+    height: 17,
+    width: "100%"
+  }
+});
+
 const CgnCardList = (props: Props) => (
-  <View>
-    {
-      // TODO Replace component with folded card when available
-      props.isCgnActive && (
-        <ButtonDefaultOpacity
-          color={IOColors.blue}
-          onPress={props.navigateToCgnDetailScreen}
-          style={{ width: "100%" }}
+  <>
+    {props.isCgnActive && (
+      <>
+        {Platform.OS === "android" && <View style={styles.upperShadowBox} />}
+        <ImageBackground
+          source={cgnBackground}
+          style={styles.preview}
+          imageStyle={styles.imagePreview}
         >
-          <H5 color={"white"}>{"GO TO CGN DETAIL"}</H5>
-        </ButtonDefaultOpacity>
-      )
-    }
-  </View>
+          <TouchableDefaultOpacity
+            onPress={props.navigateToCgnDetailScreen}
+            style={[
+              IOStyles.row,
+              styles.spaced,
+              styles.paddedContentPreview,
+              { alignItems: "center" }
+            ]}
+          >
+            <H3 color={"black"} weight={"Bold"}>
+              {TypedI18n.t("bonus.cgn.name")}
+            </H3>
+            <Image source={cgnLogo} style={styles.previewLogo} />
+          </TouchableDefaultOpacity>
+        </ImageBackground>
+      </>
+    )}
+  </>
 );
 
 const CgnCardListMemo = React.memo(
```

Comment:
Since this components loads CGN data by it self, could we remove the loading from `WalletHomeScreen`?
(https://github.com/pagopa/io-app/blob/546dbefa81970f0c16cf212228acb038d48b2afc/ts/screens/wallet/WalletHomeScreen.tsx#L233)

Now, on wallet opening, the load CGN request is fired two times